domains/apoloz.json

```
@@ -6,6 +6,6 @@
     "email": "anatoliy.poloz@gmail.com"
   },
   "record": {
-    "CNAME": "tolyod.github.io"
+    "A": "185.185.71.114"
```

A record has to be an array

```suggestion
    "A": ["185.185.71.114"]
```